installation/proc-add-custom-app-config-file-ocp.adoc

```diff
@@ -0,0 +1,86 @@
+[id="proc-add-custom-app-config-file-ocp_{context}"]
+= Adding a custom application configuration file to {ocp-short}
+
+To change the configuration of your {product} instance, you must do the following:
+
+* Add a custom application configuration file to {ocp-short} and reference it in the Custom Resource (CR). In {ocp-short}, you can use the following example as a base template to create a ConfigMap such as `app-config-rhdh.yaml`:
```

Comment:
I think it reads a little strangely to have a bullet list like this and then have it followed by a procedure - maybe integrate these bullets into the procedure below if possible instead, e.g. include the example ConfigMap as part of step 2? It currently still feels like two procedures in a module.

Reply:
@abrennan89 I agree. This content existed here before, I just reformatted it into a bulleted list to make what we were saying a little more clear. I think some of this should actually be part of the Prerequisites section. Let's address this issue in a separate Jira ticket / PR.

Reply:
https://issues.redhat.com/browse/RHIDP-2384